src/types/layers.ts

```diff
@@ -33,5 +33,6 @@ export const doesAnyOfTheseLayersSupportEncryptedTransactions = (layers: Layer[]
   uniq(layers).some(doesLayerSupportEncryptedTransactions)
 
 export const isLayerHidden = (layer: Layer): boolean => hiddenLayers.includes(layer)
+export const isLayerNotHidden = (layer: Layer): boolean => !hiddenLayers.includes(layer)
```

Comment:
Not sure we need this? `!isLayerHidden(layer)`?

Reply:
The difference between `!isLayerHidden(layer)` and `isLayerNotHidden()` is that we can use the latter as an arg for `.filter()`.

I find `layers.filter(isLayerNotHidden)` more readable than `layers.filter(layer => !isLayerHidden(layer))`

I think the extra declaration here is worth so that we can get an easier to read code elsewhere.

What do you think?

Reply:
Removed it anyway.